src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -25,73 +25,5 @@ internal static bool GetFileAttributesEx(string name, GET_FILEEX_INFO_LEVELS fil
             Debug.Assert(nameWithExtendedPrefix != null, "null not expected when non-null is passed"); // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
             return GetFileAttributesExPrivate(nameWithExtendedPrefix, fileInfoLevel, ref lpFileInformation);
         }
-
```

Comment:
The rest of the changes are just unifying interop files between CoreCLR and CoreFX.